webapp/src/components/FilterPanel.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from "react";
+
+interface FilterTagProps {
+    tag: string;
+    selected: boolean;
+    onClick: (selected: string) => void;
+}
+
+export class FilterTag extends React.Component<FilterTagProps> {
+    constructor(props: FilterTagProps) {
+        super(props)
+        this.clickHandler = this.clickHandler.bind(this);
+    }
+
+    render() {
+        return <div className="filter-tag">
+            <div className="filter-tag-box" role="checkbox" onClick={this.clickHandler} aria-checked={this.props.selected}>
+                <i className={this.props.selected ? `icon check square outline` : `icon square outline`}></i>
```

Comment:
```suggestion
                <i className={`icon square outline ${this.props.selected ? "check" : ""}`}></i>
```

nit: helps to minimize the difference so it's clear what the prop is changing